docs/data/data-grid/filtering/quick-filter.md

```diff
@@ -19,7 +19,7 @@ The quick filter values can be initialized by setting the `filter.filterModel.qu
     filter: {
       filterModel: {
         items: [],
-        quickFilterValues: ['quick', 'filter'],
+        quickFilterValues: ['ab'],
```

Comment:
Should we still somehow enforce that `quickFilterValues` is of type `string[]` and spaces in quick filter are used as delimiter to generate this array (which will somehow get hidden by this change)?

Reply:
sry for the late reply. 🙇🏼 
Maybe we could also use a more static dataset like the movie data. 
Randomized row values could potentially lead to empty results when we initialize them.

WDYT @MBilalShafi ?

Reply:
> sry for the late reply. 🙇🏼

Likewise 😄 

> Maybe we could also use a more static dataset like the movie data.

Yes, that could even be better. What I meant though was adding a callout above or below this demo which says something like:

```js
:::info
The values entered in quick filter field are delimited using space to generate `quickFilterValues` array which is used in quick filtering.

For example, if you type `quick filter` in the field, it'd be passed as `['quick', 'filter']` to the `filterModel.quickFilterValues`.
:::
```

Reply:
@MBilalShafi I did change a few more thing:
- switched to movie data for the first 3 examples
- changed description for the `excludeHiddenColumns` prop a bit due to a change in logic
- added a `valueFormatter` to the movie data `year` column since it did show it in number format `'2,017'` instead of in 'year-format' `'2017'`